### PR TITLE
Merch item enable

### PR DESCRIPTION
**Pull request by Eagerlearn**
* Three user stories combined into this request (user story 34, 33, 32)
* Added new column to Items table (:status)
* Added Items controller and hand-rolled route for the enable/disable button functionality
* Added flash message update action on the merchant items, items show page
* Added edit functionality on the merchant items, items show page

### CODE REVIEW

File: spec/features/merchant_items/index_spec.rb [hunk 1]
```diff
@@ -16,5 +16,22 @@
...
+      item_3 = merchant.items.create!(name: 'Mug', unit_price: 15, description: 'Coffee')
+      item_4 = merchant_2.items.create!(name: 'Kettle', unit_price: 20, description: 'Tea')
+
+      visit "/merchants/#{merchant.id}/items"
+      expect(page).to have_button("Disable/Enable #{item_1.name}")
+      expect(page).to have_content("available")
+
+      click_button "Disable/Enable #{item_1.name}"
+      expect(current_path).to eq("/merchants/#{merchant.id}/items")
+      expect(page).to have_content("unavailable")
```
Comment (B-gann21) [inline spec/features/merchant_items/index_spec.rb:34]: Just gotta throw in a third click_button here, to enable the item and test that it can be enabled

**Comment by CLRM1** [comment]
Looks great, let's get the coverage up to 100% for features.

**Comment by B-gann21** [comment]
Looks great!

**Comment by CLRM1** [comment]
Great work!

**Pull request merged**